src/server/metrics.rs

```diff
@@ -600,3 +608,19 @@ pub fn record_request_source_metrics(source: String, duration: Duration) {
         }
     });
 }
+
+impl From<u64> for ResourcePriority {
+    fn from(priority: u64) -> Self {
+        // the mapping definition of priority in TIDB repo,
+        // see: https://github.com/pingcap/tidb/blob/8b151114546d6a02d8250787a2a3213620e30524/parser/parser.y#L1740-L1752
+        if priority == 0 {
+            Self::medium
```

Comment:
NIT: What about treat 0 as unknown as 0 means priority is not set.

Reply:
keep same with https://github.com/tikv/tikv/blob/a0dbe2d0b893489015fc99ae73c6646f7989fe32/components/resource_control/src/resource_group.rs#L78-L90

Reply:
mapping unset priority to unknown can help us determine the number of requests that priority is not set. We can't do this for rate limiter because there is no separate limiter for unset request. So I'd like to map 0 to unknown, but to medium is also ok to me.

Reply:
yes, make sense.